src/taoensso/sente.cljx

```diff
@@ -81,6 +81,21 @@
   (:require-macros
    [cljs.core.async.macros :as asyncm :refer (go go-loop)]))
 
+; Note: requiring goog.global would be better, but currently breaks the cljs build: http://dev.clojure.org/jira/browse/CLJS-1677?page=com.atlassian.streams.streams-jira-plugin:activity-stream-issue-tab
```

Comment:
You don't need to require `goog` to get global. I'm pretty sure that `goog.global` is automatically in scope without a require.